src/makeCalculator.js

```diff
@@ -5,6 +5,29 @@
  */
 function makeCalculator() {
   // write code here
+  let result = 0;
+
+  const calculator = {
+    add: (num) => result + num,
+    subtract: (num) => result - num,
+    divide: (num) => result / num,
+    multiply: (num) => result * num,
```

Comment:
[CODE STYLE] - When performing arithmetic operations and assigning the result to a variable, use the assignment operator combined with the arithmetic operator. For instance, instead of `result + num`, use `result += num` to add `num` to `result`. This should be applied to the `add`, `subtract`, `multiply`, and `divide` methods.